### ISSUE TITLE
Update dotcms repo on OSGI plugin examples

### ISSUE THREAD

**Issue by cfalzone** [dotCMS/core#7713]
All of the plugin examples reference 3.0, which seems to have broken dependencies.  For example:  https://github.com/dotCMS/core/blob/master-3.2/docs/examples/osgi/com.dotcms.3rd.party/build.gradle#L18

Just changing it to 3.1 resolves the issue and makes the plugins buildable.

Here is the error I get when I try to deploy any OSGI plugin against the 3.0 repo:

```
BOS-CFalzone-2:dotcms.plugins.spammenot cfalzone$ gradle jar
:compileJava

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not find com.dotcms.enterprise:ee:3.0PR1_db45d51.
  Required by:
      com.aquent:dotcms-plugins-spammenot:1.2.0-SNAPSHOT > com.dotcms:dotcms:3.0
> Could not find com.dotcms.enterprise:ee:3.0PR1_db45d51.
  Required by:
      com.aquent:dotcms-plugins-spammenot:1.2.0-SNAPSHOT > com.dotcms:dotcms:3.0 > com.dotcms:starter:20141006

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 8.764 secs
```

Thanks!

**Comment by jgambarios** [comment]
Also related with: #7694

**Comment by oarrietadotcms** [comment]
PR: https://github.com/dotCMS/core/pull/7731

**Comment by bryanboza** [comment]
Fixed, now we have the 3.2 version in the artifactory
